new/LIT7058Sababata.xml

```diff
@@ -0,0 +1,79 @@
+                <title xml:lang="gez" type="normalized" corresp="#t1">Ṣababatǝna mǝdrǝki (Qǝne of mawaddǝs type)</title>
+                <title xml:lang="en" corresp="#t1">Your earth became narrow for us ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A <term key="Qene">qǝne</term>of possibly <term key="Mawaddes">mawaddǝs</term> type.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-09-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7058Sababata" passive="BNFabb145"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>The text is taken from <ref type="mss" corresp="BNFabb154"/>, fol. 56vc
+                         (division into verses is indicated with the sign ።.</note>
+                    <ab>                                                  
+                        <l n="1">ጸበበተነ፡ ምድርከ፡ በእለተ፡ እሑድ፡ እስመ፡ ፈጠርካ፡ ስፍ<supplied reason="undefined" resp="DN">ሐ</supplied><gap reason="illegible"/>፡ እግዚአብሔር፡ ፌማ።</l> 
+                        <l n="2"><supplied reason="undefined" resp="DN">እስ</supplied>መ፡ ሶቤሃ፡ ተሰምዓ፡ መዓተ፡ ኃይልከ<gap reason="illegible"/>ማ።</l>
```

Comment:
The text supplied by yourself is incorrectly indicated, it is supposed to be: `<l n="2">እስመ፡ <supplied reason="undefined" resp="DN">ሶ</supplied>ቤሃ፡ ተሰምዓ፡ መዓተ፡ ኃይልከ<gap reason="illegible"/>ማ።</l>`
Furthermore, I would read a ሌ or a ሴ, but not a ቤ. However, I cannot present a reasonable reading for this phrase. I would opt for a gap
For the gap, I would `add: unit="chars" quantity="1"` I think, there is space for only one character.

Reply:
You are rigth, I rearranged it a bit.